railties/test/commands/time_zones_test.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rails/command"
+require "rails/commands/time/time_command"
+
+class Rails::Command::TimeZonesTest < ActiveSupport::TestCase
+  setup :build_app
+  teardown :teardown_app
+
+  test "`rails time:zones` displays all time zones" do
+    result = run_time_zones_command
+    ActiveSupport::TimeZone.all.each do |zone|
+      assert_display_time_zone(zone, result)
+    end
+  end
+
+  test "`rails time:zones -OFFSET` displays time zone for UTC offset" do
+    result = run_time_zones_command("-8")
+    expected = <<~OUTPUT
+
+      * UTC -08:00 *
+      Pacific Time (US & Canada)
+      Tijuana
+
+    OUTPUT
+    assert_equal expected, result
```

Comment:
Is there a reason to assert on the exact output instead of using `assert_time_zone` (`assert_display_time_zone`)?

Reply:
This was easier to implement :)
I can change that.

Reply:
This has been changed.